src/components/atoms/NetworkName.tsx

```diff
@@ -24,10 +24,12 @@ export function NetworkIcon({ name }: { name: string }): ReactElement {
 export default function NetworkName({
   networkId,
   minimal,
+  textOnly,
```

Comment:
not sure about the need for this, it might be simpler to access `getNetworkDisplayName()` directly within `WalletNetworkSwither.tsx` if you just want to get the plain text

Reply:
right, wasn't sure if we need the icon too but if we stick witch the name only the function should do it